src/gui/tray/TalkReplyTextField.qml

```diff
@@ -0,0 +1,80 @@
+import QtQuick 2.15
+import Style 1.0
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import com.nextcloud.desktopclient 1.0
+
+Item {
+    id: root
+
+    function sendReplyMessage() {
+        if (replyMessageTextField.text === "") {
+            replyMessageTextFieldBorder.border.color = Style.errorBoxBorderColor;
+            return;
+        }
+
+        replyMessageTextFieldBorder.border.color = Style.ncBlue;
+
+        UserModel.currentUser.sendReplyMessage(model.conversationToken, replyMessageTextField.text, model.messageId);
+
+        replyMessageSent.text = replyMessageTextField.text;
+        replyMessageTextField.clear();
+        replyMessageTextField.visible = false
+    }
+
+    Text {
+        id: replyMessageSent
+        font.pixelSize: Style.topLinePixelSize
+        color: Style.menuBorder
+        visible: replyMessageSent.text !== ""
+    }
+
+    TextField {
+        id: replyMessageTextField
+
+        // TODO use Layout to manage width/height. The Layout.minimunWidth does not apply to the width set.
+        height: 38
+        width: 250
+
+        onAccepted: root.sendReplyMessage()
+
+        topPadding: 4
+        leftPadding: 12
+
+        placeholderText: qsTr("Reply to …")
+
+        background: Rectangle {
+            id: replyMessageTextFieldBorder
+            radius: 24
+            border.width: 1
+            border.color: Style.ncBlue
+        }
+
+        Button {
+            id: sendReplyMessageButton  
+            width: 32
+            height: parent.height
+            opacity: 0.8
+            flat: true
+            onClicked: root.sendReplyMessage()
+
+            icon {
+                source: "image://svgimage-custom-color/send.svg" + "/" + Style.ncBlue
+                width: 38
+                height: 38
+                color: hovered ? Style.menuBorder : Style.ncBlue
+            }
+
+            anchors {
+                right: replyMessageTextField.right
+                top: replyMessageTextField.top
+            }
+
+            ToolTip {
+                visible: hovered
```

Comment:
the resolution of the `hovered` property seems wrong as I see the tooltips when I should not
maybe try `sendReplyMessageButton.hovered` (but I am not sure)

![image](https://user-images.githubusercontent.com/168967/157985634-cc19f153-607c-4102-83ea-7557702c47b8.png)

Reply:
You are right, it worked.